tgui/packages/tgui/interfaces/IssueReporter.js

```diff
@@ -0,0 +1,226 @@
+import { Window } from '../layouts';
+import { useBackend, useLocalState } from '../backend';
+import { ButtonCheckbox } from 'tgui/components/Button';
+import { Box, Section, Button, Flex, TextArea, Input } from '../components';
+
+export const IssueReporter = (props, context) => {
+
+  const {
+    act,
+  } = useBackend(context);
+
+  const [
+    isRegression,
+    setIsRegression,
+  ] = useLocalState(context, 'isRegression', false);
+  const [
+    issueName,
+    setIssueName,
+  ] = useLocalState(context, 'issueName', '');
+  const [
+    replicationSteps,
+    setReplicationSteps,
+  ] = useLocalState(context, 'replicationSteps', '');
+  const [
+    expected,
+    setExpected,
+  ] = useLocalState(context, 'expected', '');
+  const [
+    actual,
+    setActual,
+  ] = useLocalState(context, 'actual', '');
+
+  const [
+    stage,
+    setStage,
+  ] = useLocalState(context, 'stage', 1);
+
+  let content = "The issue reporter is currently facing issues.";
+  let windowHeight = 480;
+
+  switch (stage)
+  {
+    case 1:
+      content = (<Regression
+        isRegression={isRegression}
+        setIsRegression={setIsRegression}
+        issueName={issueName}
+        setIssueName={setIssueName} />);
+      windowHeight = 294;
+      break;
+      case 2:
+        content = (<ExpectedBehaviours
+          expected={expected}
+          setExpected={setExpected}
+          actual={actual}
+          setActual={setActual} />);
+        windowHeight = 494;
+        break;
+      case 3:
+        content = (<ReplicationSteps
+          replicationSteps={replicationSteps}
+          setReplicationSteps={setReplicationSteps} />);
+        windowHeight = 362;
+        break;
+  }
+
+  return (
+    <Window
+      width={340}
+      height={windowHeight}
+      overflow="auto"
+      theme="generic"
+      title={"Issue reporter " + stage + "/3"}>
+      <Window.Content>
+        <Flex direction="column" height="100%">
+          <Flex.Item
+            grow={1}>
+            {content}
+          </Flex.Item>
+          <Flex.Item
+            alignSelf="flex-end">
+            <Section>
+              <Flex direction="row" width="100%">
+                {stage !== 1 ? (
+                <Flex.Item
+                  grow={1}>
+                  <Button
+                    fontSize={1.5}
+                    onClick={() => {
+                      if (stage === 1)
+                      {
+                        return;
+                      }
+                      setStage(stage - 1);
+                    }}>
+                    Previous
+                  </Button>
+                </Flex.Item>) : (
```

Comment:
Put the parenthesis on a new line, it looks bad otherwise.

Reply:
Styling will be handled by prettier